test/runtests.sh

```diff
@@ -77,6 +77,11 @@ user_pref("extensions.zotero.firstRunGuidance", false);
 user_pref("extensions.zotero.firstRun2", false);
 EOF
 
+
+if [ -z $IS_CYGWIN ]; then
+	echo "`MOZ_NO_REMOTE=1 NO_EM_RESTART=1 \"$FX_EXECUTABLE\" -v`"
+fi
+
```

Comment:
Yes, though I seem to have misplaced a comment. `-v` doesn't work on Windows. It just hangs the Fx process. Otherwise, it's pretty useful to know which version exactly the test is running on, since the version is not hard coded.